src/charm.py

```diff
@@ -60,6 +86,13 @@ def _on_content_cache_pebble_ready(self, event) -> None:
         self.model.unit.status = MaintenanceStatus(msg)
         self.on.config_changed.emit()
 
+    def _on_nginx_prometheus_exporter_pebble_ready(self, event) -> None:
+        """Handle content_cache_pebble_ready event and configure workload container."""
```

Comment:
Since we start to enforce standard docstrings, could worth it filling a complete docstring here even if the rest of the methods aren't compliant ?

Reply:
Of course! That will be included in my next commit